test/extreme.test.js

```diff
@@ -4,6 +4,7 @@ var test = require('tap').test
 var os = require('os')
 var writeStream = require('flush-write-stream')
 var spawn = require('child_process').spawn
```

Comment:
The only way I'm seeing that will work is to check if the passed in stream has an `fd` property or `stream._handle.fd` property. Do you agree?